python/ray/remote_function.py

```diff
@@ -21,6 +21,32 @@
 logger = logging.getLogger(__name__)
 
 
+_task_launch_hook = None
+
+
+def tune_task_launch_hook(fn, resources):
+    if "_export_bundle" in fn.function_name:  # delete this later
+        return
+    if not resources or resources == {"CPU": 0}:
+        return
+    print("TASK", fn, resources)
+    # TODO: only raise if Tune has no extra resources configured for the trial
```

Comment:
Get the current placement group and inspect the non-zero bundles to see if there's space?